core/foundation/inc/ROOT/StringUtils.hxx

```diff
@@ -22,6 +22,7 @@
 namespace ROOT {
 
 std::vector<std::string> Split(std::string_view str, std::string_view delims, bool skipEmpty = false);
+std::vector<std::string> ReadFilelist (std::string filelist);
```

Comment:
This doesn't seem fundamental enough (nor really a string utility) to go into `core/foundation/inc/ROOT/StringUtils.hxx`. Maybe make it a static function and move it into `tree/tree/src/TChain.cxx` instead?

Reply:
done